ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml

```diff
@@ -176,6 +176,21 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere-etcd-scaling
+- as: e2e-nutanix-ovn
+  cluster: build01
+  optional: true
+  run_if_changed: pkg/controllers/controlplanemachinesetgenerator/.*|.*nutanix.*
+  steps:
+    cluster_profile: nutanix
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e
+      from: stable:cluster-api-actuator-pkg
```

Comment:
I think this should be:
```
      commands: |
        make e2e-presubmit
      from: src
```

Reply:
> @thunderboltsid We have a similar PR for OpenStack which might be useful as an example on what it is normally expected for a CPMS platform integration: https://github.com/openshift/release/pull/38605/files

My understanding was that we wanted to create a optional pre-submit that we could invoke when needed. Do we also want a periodic job and a postsubmit job @JoelSpeed ?

Reply:
fixed